.appveyor.yml

```diff
@@ -1,6 +1,6 @@
 version: '{branch}.{build}'
 environment:
-  GRADLE_CLI_OPTS: '--no-daemon --console=verbose --stacktrace'
```

Comment:
Is there any reason that you remove this?

Reply:
Because we now launch Gradle twice?

Reply:
I thought each build has its own process but it's probably a bad idea because we have to pay for the cost of JVM startup for every build. Thanks!

Reply:
Each build runs in its own Docker container, so Gradle daemons are shared only among the current build tasks.

Reply:
Ah, I mean each `./gradlew` commands. Sorry for choosing the wrong word. 😄

Reply:
I added it back since we now run only a single `./gradlew` command. ProGuard emits some noisy messages but it's much less than before.